core/extensions.rs

```diff
@@ -482,28 +482,11 @@ pub struct ExtensionBuilder {
 
 impl ExtensionBuilder {
   pub fn js(&mut self, js_files: Vec<ExtensionFileSource>) -> &mut Self {
-    let js_files =
-      // TODO(bartlomieju): if we're automatically remapping here, then we should
-      // use a different result struct that `ExtensionFileSource` as it's confusing
-      // when (and why) the remapping happens.
-      js_files.into_iter().map(|file_source| ExtensionFileSource {
-        specifier: format!("ext:{}/{}", self.name, file_source.specifier),
-        code: file_source.code,
-      });
```

Comment:
Great to see this removed 👍, however it means that all examples should be updated to specify `ext:` prefix if they construct `ExtensionFileSource` manually - eg. `bench_util/benches/utf8.rs`